scvi/model/_multivi.py

```diff
@@ -0,0 +1,709 @@
+        **kwargs
+            Keyword args for :func:`scvi.utils.DifferentialComputation.get_bayes_factors`
+
+        Returns
+        -------
+        Differential accessibility DataFrame with the following columns:
+        prob_da
+            the probability of the region being differentially accessible
+        is_da_fdr
+            whether the region passes a multiple hypothesis correction procedure with the target_fdr
+            threshold
+        bayes_factor
+            Bayes Factor indicating the level of significance of the analysis
+        effect_size
+            the effect size, computed as (accessibility in population 2) - (accessibility in population 1)
+        emp_effect
+            the empirical effect, based on observed detection rates instead of the estimated accessibility
+            scores from the PeakVI model
+        est_prob1
+            the estimated probability of accessibility in population 1
+        est_prob2
+            the estimated probability of accessibility in population 2
+        emp_prob1
+            the empirical (observed) probability of accessibility in population 1
+        emp_prob2
+            the empirical (observed) probability of accessibility in population 2
+
+        """
+        adata = self._validate_anndata(adata)
+        col_names = _get_var_names_from_setup_anndata(adata)[self.n_genes :]
+        model_fn = partial(
+            self.get_accessibility_estimates, use_z_mean=False, batch_size=batch_size
+        )
+
+        # TODO check if change_fn in kwargs and raise error if so
+        def change_fn(a, b):
+            return a - b
+
+        if two_sided:
+
+            def m1_domain_fn(samples):
+                return np.abs(samples) >= delta
+
+        else:
+
+            def m1_domain_fn(samples):
+                return samples >= delta
+
+        all_stats_fn = partial(
+            scatac_raw_counts_properties,
+            var_idx=np.arange(adata.shape[1])[self.n_genes :],
+        )
+
+        result = _de_core(
+            adata=adata,
+            model_fn=model_fn,
+            groupby=groupby,
+            group1=group1,
+            group2=group2,
+            idx1=idx1,
+            idx2=idx2,
+            all_stats=all_stats,
+            all_stats_fn=all_stats_fn,
+            col_names=col_names,
+            mode=mode,
+            batchid1=batchid1,
+            batchid2=batchid2,
+            delta=delta,
+            batch_correction=batch_correction,
+            fdr=fdr_target,
+            change_fn=change_fn,
+            m1_domain_fn=m1_domain_fn,
+            silent=silent,
+            **kwargs,
+        )
+
+        # manually change the results DataFrame to fit a PeakVI differential accessibility results
+        result = pd.DataFrame(
+            {
+                "prob_da": result.proba_de,
+                "is_da_fdr": result.loc[:, "is_de_fdr_{}".format(fdr_target)],
```

Comment:
f string is good for this situation
```suggestion
                "is_da_fdr": result.loc[:, f"is_de_fdr_{fdr_target}"],
```